src/openforms/registrations/contrib/objects_api/tests/test_backend_v2.py

```diff
@@ -249,6 +254,17 @@ class V2HandlerTests(TestCase):
     to the Objects API.
     """
 
+    BASE_REGISTRATION_OPTIONS: RegistrationOptionsV2 = {
+        "version": 2,
+        "objecttype": "-dummy-",
+        "objecttype_version": 1,
+        "upload_submission_csv": False,
+        "informatieobjecttype_submission_report": "",
+        "informatieobjecttype_attachment": "",
+        "organisatie_rsin": "000000000",
+        "variables_mapping": [],
+    }
```

Comment:
I don't like this indirectness, and you always end up with new tests that need to modify this constant which then causes other tests to break or become too bloated. Please just specify just the necessary configuration for each test.

If it's really irrelevant, use a factory, but at least isolate tests from each other.